presto-teradata-functions/src/main/java/com/facebook/presto/teradata/functions/TeradataStringFunctions.java

```diff
@@ -38,6 +39,24 @@ public static long index(@SqlType(StandardTypes.VARCHAR) Slice string, @SqlType(
         return StringFunctions.stringPosition(string, substring);
     }
 
+    @Description("suffix starting at given index")
+    @ScalarFunction
+    @LiteralParameters("x")
+    @SqlType("varchar(x)")
+    public static Slice substring(@SqlType("varchar(x)") Slice utf8, @SqlType(StandardTypes.BIGINT) long start)
+    {
+        return StringFunctions.substr(utf8, start);
```

Comment:
It does not matter much, I kept the convention started with Teradata Functions (which contain other aliases). There is already `substr` function so why pollute core Presto with aliases not used by anyone but Teradata users? If it was a new function that would be okay to introduce it but just for an alias? I believe it's better to keep it in a plugin.

I don't have strong opinions, I am open for both approaches, so let's wait for different opinions.